README.md

```diff
@@ -10,6 +10,10 @@ Shift-left API design conformance checks into Visual Studio Code with integrated
 
 ![linting](./media/integrated-lint.png)
 
+Ensure that new API versions don't break API consumers with breaking change detection support, powered by Optic.
+
+![image](https://github.com/pierceboggan/vscode-azureapicenter-1/assets/1091304/78777fd4-f7cd-488e-a152-20d0f696fa8a)

```

Comment:
now the `api-center` is open source, so please upload the image to `media` folder and reference from it.